integration-tests-scala/src/test/scala/tech/beshu/ror/integration/suites/base/BaseAdminApiSuite.scala

```diff
@@ -14,35 +14,47 @@
  *    You should have received a copy of the GNU General Public License
  *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
  */
-package tech.beshu.ror.integration.plugin.base
+package tech.beshu.ror.integration.suites.base
 
-import com.dimafeng.testcontainers.{ForAllTestContainer, MultipleContainers}
+import cats.data.NonEmptyList
+import com.dimafeng.testcontainers.MultipleContainers
 import org.apache.commons.lang.StringEscapeUtils.escapeJava
 import org.scalatest.Matchers._
 import org.scalatest.{BeforeAndAfterEach, WordSpec}
-import tech.beshu.ror.utils.containers.{ElasticsearchNodeDataInitializer, ReadonlyRestEsClusterContainer}
+import tech.beshu.ror.integration.suites.base.support.{BaseIntegrationTest, MultipleClientsSupport}
+import tech.beshu.ror.utils.containers.generic._
 import tech.beshu.ror.utils.elasticsearch.{ActionManagerJ, DocumentManagerJ, IndexManagerJ, SearchManager}
 import tech.beshu.ror.utils.httpclient.RestClient
 import tech.beshu.ror.utils.misc.Resources.getResourceContent
 
-trait BaseAdminApiTests extends WordSpec with ForAllTestContainer with BeforeAndAfterEach {
+trait BaseAdminApiSuite
+  extends WordSpec
+    with BeforeAndAfterEach
+    with BaseIntegrationTest
+    with MultipleClientsSupport {
+  this: EsContainerCreator =>
 
   protected def readonlyrestIndexName: String
+  protected def rorWithIndexConfig: EsClusterContainer
+  protected def rorWithNoIndexConfig: EsClusterContainer
 
-  protected def rorWithIndexConfig: ReadonlyRestEsClusterContainer
-  protected def rorWithNoIndexConfig: ReadonlyRestEsClusterContainer
 
-  override lazy val container: MultipleContainers = MultipleContainers(rorWithIndexConfig, rorWithNoIndexConfig)
+  private lazy val ror1_1Node = rorWithIndexConfig.nodesContainers.head
+  private lazy val ror1_2Node = rorWithIndexConfig.nodesContainers.tail.head
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name 'ror1_2Node' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=5163287)